### PR TITLE
CLC-6773 CLC-6668 Switch OEC's source for person name & email

### PR DESCRIPTION
**Pull request by raydavis**
https://jira.ets.berkeley.edu/jira/browse/CLC-6668

The code could be refactored, but leveraging what was already there makes the development job faster and safer. In this case, I chose to err on the side of expediency. Reviewers should feel free to push back if they disagree.

### CODE REVIEW

**Comment by paulkerschen** [comment]
I approve the surgical strike, @raydavis. Waiting for T...

**Comment by raydavis** [comment]
Apologies, @paulkerschen -- on self-review, I noticed a now redundant query that begged to be deleted. I've amended the PR.

File: app/models/edo_oracle/adapters/oec.rb [hunk 1]
```diff
@@ -41,29 +41,19 @@ def self.supplement_user_attributes(rows, columns=nil)
...
 
-        User::BasicAttributes.attributes_for_uids(rows_without_email.keys).each do |attrs|
-          rows_without_email[attrs[:ldap_uid]].each do |row|
+        User::BasicAttributes.attributes_for_uids(rows_missing_attributes.keys).each do |attrs|
+          rows_missing_attributes[attrs[:ldap_uid]].each do |row|
             row[email_address_key] = attrs[:email_address]
-          end
-        end
-
-        User::BasicAttributes.attributes_for_uids(rows_without_names.keys).each do |attrs|
```
Comment (raydavis) [inline app/models/edo_oracle/adapters/oec.rb]: This is the excess query that I trimmed.

**Comment by paulkerschen** [comment]
Leaner!

**Pull request merged**